common/reviews/api/heft.api.md

```diff
@@ -325,6 +335,11 @@ export interface IMetricsData {
     taskTotalExecutionMs: number;
 }
 
+// @beta (undocumented)
```

Comment:
Add the missing docs.

Reply:
(We need to work to eliminate all these `(undocumented)` items in what is a rather complicated set of APIs.)

Reply:
Added docs.